sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -680,3 +680,23 @@ ALTER TABLE `pnotes` ADD `update_date` DATETIME DEFAULT NULL;
 #IfNotColumnType onsite_documents full_document MEDIUMBLOB
 ALTER TABLE `onsite_documents` CHANGE `full_document` `full_document` MEDIUMBLOB;
 #EndIf
+
+#IfMissingColumn mail_street facility
```

Comment:
This line should be:
```
#IfMissingColumn facility mail_street
```

Reply:
Also need to fix the other #IfMissingColumn lines below

Reply:
fixed them all